org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/extensions/contentmodel/participants/XMLSyntaxErrorCode.java

```diff
@@ -47,7 +47,7 @@
 	EqRequiredInAttribute, the_element_type_lmsg("the-element-type-lmsg"), EqRequiredInXMLDecl, IllegalQName,
 	InvalidCommentStart, LessthanInAttValue, MarkupEntityMismatch, MarkupNotRecognizedInContent,
 	NameRequiredInReference, OpenQuoteExpected, PITargetRequired, PseudoAttrNameExpected, QuoteRequiredInXMLDecl,
-	SDDeclInvalid, SpaceRequiredBeforeEncodingInXMLDecl, SpaceRequiredBeforeStandalone, SpaceRequiredInPI,
+	SDDeclInvalid, src_element_3("src-element.3"), SpaceRequiredBeforeEncodingInXMLDecl, SpaceRequiredBeforeStandalone, SpaceRequiredInPI,

```

Comment:
For me it's not a syntax error but an error coming from XML Schema. In other words please remove src_element_3 in XMLSchemaErrorCode (error in XML file coming from a XML Schema)

Reply:
Did you mean remove src_element_3 in XMLSyntaxErrorCode instead?

Reply:
yes sorry and move to XMLSchemaErrorCode

Reply:
Awesome. I also moved my test case from XMLSyntaxDiagnosticsTest to XMLSyntaxDiagnosticsTest